MANUAL.txt

```diff
@@ -2910,10 +2910,12 @@ legal (though ugly) pipe table:
 The cells of pipe tables cannot contain block elements like paragraphs
 and lists, and cannot span multiple lines.  If a pipe table contains a
 row whose printable content is wider than the column width (see
-`--columns`), then the cell contents will wrap, with the
-relative cell widths determined by the widths of the separator
-lines.  (In this case, the table will take up the full text
-width.)  If no lines are wider than column width, then
+`--columns`), then the table will take up the full text width and
+the cell contents will wrap, with the relative cell widths determined
+by the number of dashes in the line separating the table header from
+the table body (for example `---|-` would make the first column ¾
```

Comment:
To make this sentence a bit less long, why not make the parenthesized part a separate,
parenthesized sentence?  Also, I think it's safer to use 3/4 and 1/4 rather than the unicode
fractions -- that way when we render the man page we don't have to count on a font
with these glyphs.

Reply:
👍🏽that's changed.